lib/conjur/base.rb

```diff
@@ -169,6 +169,13 @@ def initialize username, api_key, token, remote_ip = nil
       @username = username
       @api_key = api_key
       @token = token
+      
+      # Assume that the token was recently minted, log a warning if not
+      @token_born = gettime   
+      if token && (delta = Time.now - Time.parse(token['timestamp'])) > 10
+        $stderr.puts "WARNING: token timestamp offset by #{delta.to_i} seconds"
+      end
+
```

Comment:
One way would be to abstract `gettime` to a class that represents a timestamp along with the clocksource (RTC vs. monotonic). The class could have a `age` member that would do the right thing.